src/MultiplicationTable.ts

```diff
@@ -1,5 +1,49 @@
 export class MultiplicationTable {
   public render(start: number, end: number): string{
-    return '1*1=1'
+    if(this.isNumberPairValid(start,end)){
+      console.log(this.CreateMutiplicationTable(start,end));
+      return this.CreateMutiplicationTable(start,end)
+    }
+    else{
+      return "";
+    }
+  }
+
+  public isNumberPairValid( start: number, end: number): boolean{
+    return( this.isNumberInRange10(start) && this.isNumberInRange10(end) && this.isStartLessEqualThanEnd(start,end))
+
+  }
+
+  private isNumberInRange10(num : number){
+    if (num <= 10 && num > 0 ){
+      return true;
+    }
+    return false;
+  }
+
+  private isStartLessEqualThanEnd( start: number, end: number){
+    if ( start <= end){
+      return true;
+    }
+    return false;
+  }
+
+  public CreateMutiplicationTable(start: number, end: number): string{
+    if (start === end){
+      return this.BuildOneLineMultiplication(start, end)
+    }
+    else{
+      return (this.CreateMutiplicationTable(start, end-1) + this.BuildOneLineMultiplication(start,end))
```

Comment:
In the last line, it also has '\n'.  Is it feasible to do recursive  from the first line to the last line to avoid a '\n' in the last line?